actors/common/src/main/java/org/sunbird/learner/actors/InitialisationActor.java

```diff
@@ -0,0 +1,265 @@
+      }
+    }
+    return true;
+  }
+  private Map<String, Object> elasticSearchComplexSearch(
+    Map<String, Object> filters, String index, String type) {
+
+  SearchDTO searchDTO = new SearchDTO();
+  searchDTO.getAdditionalProperties().put(JsonKey.FILTERS, filters);
+
+  return ElasticSearchUtil.complexSearch(searchDTO, index, type);
+}
+
+private boolean isSlugUnique(String slug) {
+  if (!StringUtils.isBlank(slug)) {
+    Map<String, Object> filters = new HashMap<>();
+    filters.put(JsonKey.SLUG, slug);
+    filters.put(JsonKey.IS_ROOT_ORG, true);
+    Map<String, Object> esResult =
+        elasticSearchComplexSearch(
+            filters, EsIndex.sunbird.getIndexName(), EsType.organisation.getTypeName());
+    if (isNotNull(esResult)
+        && esResult.containsKey(JsonKey.CONTENT)
+        && isNotNull(esResult.get(JsonKey.CONTENT))) {
+      return (((List) esResult.get(JsonKey.CONTENT)).isEmpty());
+    }
+  }
+  return false;
+}
+
+
+private String validateHashTagId(String hashTagId, String opType, String orgId) {
+  Map<String, Object> filters = new HashMap<>();
+  filters.put(JsonKey.HASHTAGID, hashTagId);
+  SearchDTO searchDto = new SearchDTO();
+  searchDto.getAdditionalProperties().put(JsonKey.FILTERS, filters);
+  Map<String, Object> result =
+      ElasticSearchUtil.complexSearch(
+          searchDto,
+          ProjectUtil.EsIndex.sunbird.getIndexName(),
+          ProjectUtil.EsType.organisation.getTypeName());
+  List<Map<String, Object>> dataMapList = (List<Map<String, Object>>) result.get(JsonKey.CONTENT);
+  if (opType.equalsIgnoreCase(JsonKey.CREATE)) {
+    if (!dataMapList.isEmpty()) {
+      throw new ProjectCommonException(
+          ResponseCode.invalidHashTagId.getErrorCode(),
+          ResponseCode.invalidHashTagId.getErrorMessage(),
+          ResponseCode.CLIENT_ERROR.getResponseCode());
+    }
+  } else if (opType.equalsIgnoreCase(JsonKey.UPDATE) && !dataMapList.isEmpty()) {
+    Map<String, Object> orgMap = dataMapList.get(0);
+    if (!(((String) orgMap.get(JsonKey.ID)).equalsIgnoreCase(orgId))) {
+      throw new ProjectCommonException(
+          ResponseCode.invalidHashTagId.getErrorCode(),
+          ResponseCode.invalidHashTagId.getErrorMessage(),
+          ResponseCode.CLIENT_ERROR.getResponseCode());
+    }
+  }
+  return hashTagId;
+}
+
+
+private void addInitialisationFlagToSystemSettings() {
+  Map<String, Object> map = new HashMap<>();
+  map.put(JsonKey.ID, JsonKey.IS_INITIALISED);
+  map.put(JsonKey.FIELD,  JsonKey.IS_INITIALISED);
+  map.put(JsonKey.VALUE, String.valueOf(true));
+  Response response = cassandraOperation.insertRecord(JsonKey.SUNBIRD, JsonKey.SYSTEM_SETTINGS_DB, map);
+  ProjectLogger.log(
+      "Insert operation result for initialised status =  "
+          + response.getResult().get(JsonKey.RESPONSE),LoggerEnum.INFO.name());
+}
+
+private void checkIfAlreadyInitialised(){
+  ProjectLogger.log("get initSetting record started" , LoggerEnum.INFO.name());
```

Comment:
Change log and level to DEBUG:
"SystemInitActor: checkIfRootOrgAlreadyInitialised called"